### DISCUSSION TITLE
Serialisation of Object

### DISCUSSION THREAD

**Discussion by ayushkhanduri** [pallets/flask#4204] [Q&A]
I have an object that I am sending as response in flask 

```
class LyricsConfigType:
    line: str
    font_size: str
    color: str
    background_color: str
    position: Position = Position()

class PageConfigType:
    page: int
    lyrics_config: List[LyricsConfigType] = []

class Position:
    x_axis: str
    y_axis: str

```

I am sending the object of List[PageConfigType]. I want to serialise this object

**Comment by davidism** [comment]
Use a serialization library such as Marshmallow or Pydantic.

**Reply by ayushkhanduri** [reply]
I mean I do understand that , I saw marshmallow's api, it basically says to create a schema for each an every class and load your values in those Schemas, I mean isn't there an easier way to do ?

**Reply by davidism** [reply]
No, this isn't something Flask provides.
